docs/ops/sort/MatrixNMS_8.md

```diff
@@ -0,0 +1,156 @@
+##  MatrixNms<a name="MatrixNms"></a> {#openvino_docs_ops_sort_MatrixNms_8}
```

Comment:
Update `ie_docs.xml` file as in the https://github.com/openvinotoolkit/openvino/pull/5773